libr/core/cmd_anal.c

```diff
@@ -5173,7 +5177,20 @@ static void cmd_anal_syscall(RCore *core, const char *input) {
 		// JSON support
 		break;
 	case '\0':
-		cmd_syscall_do (core, -1); //n);
+		if (cmd_anal_isThumb (core)) {
```

Comment:
i dont like this conditional in here

Reply:
I dont know how to action that comment

Reply:
You can do this inside cmd_syscall_do without messing up the caller code